devel/fetch_metrics/fetch_metrics.py

```diff
@@ -201,10 +202,15 @@ def output_results(comparison_list):
             comparison_object['quality_metric'] = quality_job.result.return_value
             comparison_object['ground_truth_comparison'] = ground_truth_comparison_job.result.return_value
             continue
-        if quality_job.status == 'error' or ground_truth_comparison_job.status == 'error':
-            print(f"ERROR: comparison {extract_sorting_reference_name(comparison_object)} had an errored job.")
-        elif quality_job.status != 'finished' or ground_truth_comparison_job.status != 'finished':
-            print(f"ERROR: unfinished non-errored job in {extract_sorting_reference_name(comparison_object)}--possible hither error")
+        if quality_job.status == 'error':
```

Comment:
adjust how these errors are reported. Use "WARNING" instead of error... because the process does not halt.